pkg/dnshelper/dnshelper.go

```diff
@@ -0,0 +1,46 @@
+package dnshelper
+
+import (
+	"fmt"
+	"html/template"
+
+	"github.com/massalabs/station/pkg/convert"
+	"github.com/massalabs/station/pkg/node"
+)
+
+const (
+	metaKey = "META"
+
+	// Indexes of data in website name key.
+	indexOfWebsiteAddress     = 0 // Index of website Address in the dnsValue array
+	indexOfWebsiteDescription = 2 // Index of website Description in the dnsValue array
+)
+
+// Helper function to fetch website values from DNS entry.
+func GetWebsiteValues(dnsValue []byte) (string, string, error) {
+	websiteValue := convert.ByteToStringArray(dnsValue)
+	if len(websiteValue) <= indexOfWebsiteAddress {
+		return "", "", fmt.Errorf("invalid website value: missing website address")
+	}
+
+	websiteStorerAddress := websiteValue[indexOfWebsiteAddress]
```

Comment:
Same, we alreadu know that the this is a DNS for a website. you can simply the varaible name:
```suggestion
	address := websiteValue[indexOfWebsiteAddress]
```